src/Handlers/PaymentResponseHandler.php

```diff
@@ -268,6 +267,7 @@ function (Context $context) use ($orderTransactionId, $customFields) {
             case self::CHALLENGE_SHOPPER:
             case self::RECEIVED:
             case self::PRESENT_TO_SHOPPER:
+            case self::PENDING:
```

Comment:
This is because for paypal, we make the call to `/payments` and handle the response before loading the popup window, and setting the order with a `Pending` resultCode to paid here would 
break the cancel/error flow where we try to set the payment as failed.
The `AUTHORISATION` notification would set the payment to paid